app/styles/modules/_tooltip.scss

```diff
@@ -15,6 +15,9 @@
     right: 3px;
   }
 
+  a {
+    color: #fff;
```

Comment:
We dont yet have the $color-white variable (thats in #3753), but I can set it to some other variable which has white in it. `$message-text-color` seems like a good choice.